README.md

```diff
@@ -11,7 +11,7 @@ Ansible modules which provide different ManageIQ operations.
 
 ManageIQ Python API Client package [manageiq-api-client-python] (https://github.com/ManageIQ/manageiq-api-client-python/).
 
-    `$ pip install git+https://github.com/ManageIQ/manageiq-api-client-python.git`
+    `$ pip install manageiq-client`
```

Comment:
I left line #14 as is cause this part only refers only to client and the `requirements.txt` may also include other packages.

changed `[manageiq-api-client-python]` to `[manageiq-client]`